src/views/projects/SourceBrowser/FileDiff.svelte

```diff
@@ -120,11 +278,37 @@
     color: var(--color-negative-6);
     background-color: var(--color-negative-2);
   }
+  .diff-line[data-type="@"] > * {
+    color: var(--color-foreground-5);
+    background-color: var(--color-foreground);
+  }
+  .diff-line[data-type="@+"] > * {
+    color: var(--color-positive);
+    background-color: var(--color-positive-6);
+  }
+  .diff-line[data-type="@-"] > * {
+    color: var(--color-negative);
+    background-color: var(--color-negative-6);
+  }
+  .diff-line[data-type="@"] {
+    border-left: 4px solid gold;
+  }
+  .diff-line[data-type="@+"] {
+    border-left: 4px solid gold;
```

Comment:
had to use `gold` color, cause `--color-caution` in not what I want.

Reply:
We should use the primary color for now to avoid introducing colors that aren't part of the theme. We use the primary color for text selection at the moment anyway.


<img width="361" alt="Screenshot 2023-07-19 at 16 45 34" src="https://github.com/radicle-dev/radicle-interface/assets/158411/3f3e6734-8a20-4a49-83c4-9ca0d9fb03c5">



```diff
diff --git a/src/views/projects/SourceBrowser/FileDiff.svelte b/src/views/projects/SourceBrowser/FileDiff.svelte
index 6aca391..ab40256 100644
--- a/src/views/projects/SourceBrowser/FileDiff.svelte
+++ b/src/views/projects/SourceBrowser/FileDiff.svelte
@@ -293,16 +293,16 @@
     background-color: var(--color-negative-6);
   }
   .diff-line[data-line-type="@"] {
-    border-left: 4px solid gold;
+    border-left: 4px solid var(--color-primary);
   }
   .diff-line[data-line-type="@+"] {
-    border-left: 4px solid gold;
+    border-left: 4px solid var(--color-primary);
   }
   .diff-line[data-line-type="@-"] {
-    border-left: 4px solid gold;
+    border-left: 4px solid var(--color-primary);
   }
   .diff-line[data-line-type="@@@"] {
-    border-left: 4px solid gold;
+    border-left: 4px solid var(--color-primary);
     background-color: var(--color-foreground-4);
   }
   .diff-line-number {
```

Reply:
Actually, let's do this, it's a bit more in line with the current overall design:

```diff
diff --git a/src/views/projects/SourceBrowser/FileDiff.svelte b/src/views/projects/SourceBrowser/FileDiff.svelte
index 6aca391..8e5fcae 100644
--- a/src/views/projects/SourceBrowser/FileDiff.svelte
+++ b/src/views/projects/SourceBrowser/FileDiff.svelte
@@ -281,28 +281,28 @@
     background-color: var(--color-negative-2);
   }
   .diff-line[data-line-type="@"] > * {
-    color: var(--color-foreground-5);
-    background-color: var(--color-foreground);
+    color: var(--color-foreground-6);
+    background-color: var(--color-foreground-4);
   }
   .diff-line[data-line-type="@+"] > * {
-    color: var(--color-positive);
-    background-color: var(--color-positive-6);
+    color: var(--color-positive-6);
+    background-color: var(--color-positive-4);
   }
   .diff-line[data-line-type="@-"] > * {
-    color: var(--color-negative);
-    background-color: var(--color-negative-6);
+    color: var(--color-negative-6);
+    background-color: var(--color-negative-4);
   }
   .diff-line[data-line-type="@"] {
-    border-left: 4px solid gold;
+    border-left: 4px solid var(--color-primary);
   }
   .diff-line[data-line-type="@+"] {
-    border-left: 4px solid gold;
+    border-left: 4px solid var(--color-primary);
   }
   .diff-line[data-line-type="@-"] {
-    border-left: 4px solid gold;
+    border-left: 4px solid var(--color-primary);
   }
   .diff-line[data-line-type="@@@"] {
-    border-left: 4px solid gold;
+    border-left: 4px solid var(--color-primary);
     background-color: var(--color-foreground-4);
   }
   .diff-line-number {
```

<img width="1312" alt="Screenshot 2023-07-20 at 10 06 39" src="https://github.com/radicle-dev/radicle-interface/assets/158411/74df56cc-e1a1-4c8f-8189-1c14d21b2694">
<img width="1347" alt="Screenshot 2023-07-20 at 10 07 03" src="https://github.com/radicle-dev/radicle-interface/assets/158411/3e8f1f46-b972-499e-b92f-b06ef58b400e">

Reply:
https://github.com/radicle-dev/radicle-interface/pull/942/commits/a9d75a3b1fa093f215d536d106aa863f0bacb883